src/lookup-input.js

```diff
@@ -3,8 +3,14 @@ import React, {PropTypes as t} from 'react';
 type LookupInputPropsType = {
```

Comment:
If you want, you can shorten `LookupInputPropsType` to `PropsType`.
This type is local to this file, so it won't conflict with `PropsType` in other files.